test/e2e/storage/testsuites/api_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
2019

Reply:
Same here - new PR?

But speaking of the copyright year, is there any process in place for ensuring that this gets updated for existing files? Or is the rule "the first PR to touch an existing file must bump to or add the new year"?

Reply:
I believe the year should be when the file was created, not last modified

Reply:
Let's just make the changes now. I already approved so final lgtm will be quick